src/components/OSCALLoader.js

```diff
@@ -192,12 +224,31 @@ export default function OSCALLoader(props) {
   if (!isLoaded) {
     result = <CircularProgress />;
   } else if (oscalUrl) {
-    result = props.renderer(
-      isRestMode,
-      oscalData,
-      oscalUrl,
-      onResolutionComplete,
-      handleRestPatch
+    result = isRestMode ? (
+      <Split className={classes.split} minSize={500} sizes={[50, 50]}>
```

Comment:
Can we start this out closer to a 1/3 source and 2/3 rendered object rather than a 50/50 split?

And perhaps a `minSize` of `300`?

Reply:
Updated the values in https://github.com/EasyDynamics/oscal-react-library/pull/265/commits/298c17f713c60b046dd7c8aae7564227dd4c14ff